dev/com.ibm.ws.wsbytebuffer/bnd.bnd

```diff
@@ -0,0 +1,72 @@
+#*******************************************************************************
+# Copyright (c) 2021 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Name: WsByteBuffer
+Bundle-SymbolicName: com.ibm.ws.wsbytebuffer; singleton:=true
+Bundle-Description: Provides WsByteBuffer buffer functionality
+
+WS-TraceGroup: ChannelFramework
+
+IBM-Default-Config: OSGI-INF/wlp/defaultInstances.xml
+
+# For each exported package, create (in that package) a package-info.java
+# file, and place an @version javadoc tag in package-level javadoc. 
+# Append ";provide:=true" if this bundle also provides an implementation
+# for the exported API.
+#	com.ibm.wsspi.bytebuffer;version="1.0";-split-package:=merge-last
+
+Export-Package: com.ibm.wsspi.bytebuffer;version="1.0";provide:=true, \
+    com.ibm.ws.bytebuffer.internal;version="1.0";provide:=true, \
+    com.ibm.ws.bytebuffer.internal.resources;version="1.0";provide:=true, \
+	com.ibm.websphere.wsbytebuffer.osgi;provide:=true
+
+#	com.ibm.wsspi.bytebuffer2;version="1.0";provide:=true
+#    com.ibm.ws.bytebuffer.internal;version="1.0";provide:=true, \
```

Comment:
This internal package is needed to load the classes
ByteBufferConfiguration
FastSyncHashTable
WsByteBufferImpl
FCWsByteBufferImpl
WsByteBufferPoolManagerImpl

It makes sense to abstract these to allow buffer implementation switching so partitioning the package renaming
the externally accessed classes to not be 'internal' is just the first step.

Reply:
Discussed at the scrum, we will leave these as is for now.